applications/asset_tracker_v2/src/ext_sensors/Kconfig

```diff
@@ -9,6 +9,14 @@ menuconfig EXTERNAL_SENSORS
 
 if EXTERNAL_SENSORS
 
+config EXTERNAL_SENSORS_SET_ACCELEROMETER_TRIGGER_ON_INIT
```

Comment:
Yeah. I was thinking that too. It may be to vague the way it is and its not really clear what this means in the context of the applications behavior. Ill use ```EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO```.